tests/data_validation/data_validation_list_component.test.ts

```diff
@@ -54,8 +53,9 @@ describe("Edit criterion in side panel", () => {
         values: ["ok", "hello", "okay"],
         displayStyle: "arrow",
       });
-      ({ fixture } = await mountDataValidationPanel(model));
-      await click(fixture, ".o-dv-preview");
+      ({ fixture, env } = await mountSpreadsheet({ model }));
```

Comment:
you can mount the component SidePanels instead, it's lighter than mounting the whols=e spreadsheet